### PR TITLE
Improved pentaho-data-integration

### PR DESCRIPTION
**Pull request by edgardmessias**
* Added description
* Added shortcut
* Use checksum file instead sourceforge
* Added missing bin: Carte, Kitchen,  Pan
* Use 7-zip to extract (unzip with windows raises errors)

### CODE REVIEW

File: bucket/pentaho-data-integration.json [hunk 1]
```diff
@@ -1,19 +1,35 @@
...
+        [
+            "Spoon.bat",
+            "Pentaho Data Integration (PDI)",
+            "",
+            "spoon.ico"
+        ]
+    ],
     "checkver": {
         "url": "https://sourceforge.net/projects/pentaho/rss?path=/",
         "re": "Pentaho (?<short>[\\d.]+)/client-tools/pdi-ce-(?<version>[\\d\\w.-]+).zip"
```
Comment (Ash258) [inline bucket/pentaho-data-integration.json]:
```suggestion
        "regex": "Pentaho (?<short>[\\d.]+)/client-tools/pdi-ce-(?<version>[\\w.-]+)\\.zip"
```

File: bucket/pentaho-data-integration.json [hunk 2]
```diff
@@ -1,19 +1,36 @@
...
+            "Spoon.bat",
+            "Pentaho Data Integration (PDI)",
+            "",
+            "spoon.ico"
+        ]
+    ],
     "checkver": {
         "url": "https://sourceforge.net/projects/pentaho/rss?path=/",
-        "re": "Pentaho (?<short>[\\d.]+)/client-tools/pdi-ce-(?<version>[\\d\\w.-]+).zip"
+        "regex": "Pentaho (?<short>[\\d.]+)/client-tools/pdi-ce-(?<version>[\\w.-]+)\\.zip"
```
Comment (Ash258) [inline bucket/pentaho-data-integration.json]:
```suggestion
        "regex": "Pentaho\\s+[\\d.]+/client-tools/pdi-ce-([\\w.-]+)\\.zip"
```

File: bucket/pentaho-data-integration.json [hunk 3]
```diff
@@ -1,19 +1,36 @@
...
     },
     "autoupdate": {
-        "url": "https://downloads.sourceforge.net/project/pentaho/Pentaho%20$matchShort/client-tools/pdi-ce-$version.zip"
+        "url": "https://downloads.sourceforge.net/project/pentaho/Pentaho%20$matchShort/client-tools/pdi-ce-$version.zip#dl.7z",
```
Comment (Ash258) [inline bucket/pentaho-data-integration.json]:
```suggestion
        "url": "https://downloads.sourceforge.net/project/pentaho/Pentaho%20$majorVersion.$minorVersion/client-tools/pdi-ce-$version.zip#dl.7z",
```

**Comment by Ash258** [comment]
Also run checkver since, ther is version 8.3 in sourceforge

**Comment by edgardmessias** [comment]
Closed to favor of https://github.com/edgardmessias/scoop-pentaho

**Pull request closed without merging**